test/end-to-end/core.sh

```diff
@@ -326,14 +326,22 @@ os::cmd::expect_success 'oc whoami'
 
 echo "[INFO] Running a CLI command in a container using the service account"
 os::cmd::expect_success 'oc policy add-role-to-user view -z default'
-oc run cli-with-token --attach --image="openshift/origin:${TAG}" --restart=Never -- cli status --loglevel=4 > "${LOG_DIR}/cli-with-token.log" 2>&1
```

Comment:
--attach makes the command watch for events until it notices that the pod is running and then it attaches to the container. This is not a workaround because --attach is not working, but because of the pod never transitioning to Running (https://github.com/openshift/origin/issues/11016).